frontend/src/components/forms/UpsertServiceCall.tsx

```diff
@@ -248,9 +282,39 @@ export default function CreateServiceCallModal({
       </DialogTitle>
       <DialogContent>
         <Grid container spacing={2} paddingTop={1}>
+          <Grid item xs={4}>
+            {employees ? (
+              <UserSearchBar
+                userList={employees ?? []}
+                selectedUser={employee}
+                handleUserSelected={(_, user) =>
+                  user?.id && setValue('employeeId', user.id)
+                }
+                label='Employee'
+              />
+            ) : (
+              <Box>Loading...</Box>
+            )}
+          </Grid>
+          <Grid item xs={4}>
+            <TextField
+              value={tank?.description ?? 'Loading...'}
+              disabled
+              fullWidth
+            />
+          </Grid>
           {fields.map((field) => (
             <CreateForm key={field.name} control={control} {...field} />
           ))}
+          <Grid item xs={4}>
+            <Button
+              onClick={(e) => checkAllCeckboxes(true)}
```

Comment:
suggestion: 
be able to toggle true and false with usestate to check all boxes and uncheck all boxes

Reply:
yeah this is a good iea. https://tanknicians.atlassian.net/browse/TANK-290

Reply:
yeah this is a good iea. https://tanknicians.atlassian.net/browse/TANK-290